apps/src/components/app-sidebar.tsx

```diff
@@ -104,6 +76,8 @@ export function AppSidebar() {
 	const about_url = INTERNAL_URLS[`about-data-${currentLocale}`] || '';
 	const support_url = INTERNAL_URLS[`support-${currentLocale}`] || '';
 
+	const isSeasonalDecadal = climateVariable?.getClass() === 'SeasonalDecadalClimateVariable'
```

Comment:
Following up on [this comment](https://github.com/CanadianClimateDataPortal/climatedata-wp-theme/pull/617#discussion_r2394645973) regarding use of "`instanceof ...` instead of using `.getId().startsWith('s2d_')` or equivalent.

We have a method `.getClass()` that has the string name of the class.

That's probably what we want.

Reply:
If I try the following, look what's happening.

```diff
- const isSeasonalDecadal = climateVariable?.getClass() === 'SeasonalDecadalClimateVariable'
+ const isSeasonalDecadal = climateVariable instanceof SeasonalDecadalClimateVariable
+ const className = climateVariable?.getClass()
+
+ console.log(`climateVariable instanceof SeasonalDecadalClimateVariable counter at ${counter}`, isSeasonalDecadal)
+ console.log(`climateVariable?.getClass() counter at ${counter}`, className)
+
+ ++counter;
```

Gives:

<img width="726" height="368" alt="image" src="https://github.com/user-attachments/assets/568c1b3d-396c-4897-bd63-1c2c27171bc8" />